Mobile/lib/Components/TaskList.dart

```diff
@@ -146,6 +139,7 @@ class _TasklistState extends State<Tasklist> {
                             task: task,
                             onUpdateTask: updateTask,
                             onDeleteTask: _deleteTask,
+                            currentListType: widget.listType,
```

Comment:
Igen skal du bruge den?

Reply:
Har brug for tasklist type i task dialogen for at kunne vise de korrekte knapper baseret på hvilken liste tasken vises på.